### DISCUSSION TITLE
Announcement: Smart Connections Community "Lean Coffee" Virtual Meeting - Friday, August 2nd, 12pm-1pm EDT (UTC-4)

### DISCUSSION THREAD

**Discussion by mmoatts** [brianpetro/obsidian-smart-connections#708] [Announcements]
Hi everyone!

We have our first Smart Connections community "[Lean Coffee](https://www.perplexity.ai/search/what-is-a-lean-coffee-ptnspQzMR3igs9h50QWHcg)" meeting coming up this Friday! Sign up below:

https://lu.ma/jyyap4tq

**Event Information:**
This is a Smart Connections community event hosted the 1st Friday of every month focused on **Non-Technical Scope** including:

1. Use case exploration, including show & tell
2. Best practices for existing features
3. Opportunities for community improvements
4. Conceptual ideation for new features

To best serve the whole community, the events are recorded for those who can't make it. We'll also have a community virtual white board and AI-assisted summaries made available as well.

**_Important Request:_** These meetings are intended to build community, exchange ideas, and help grow understanding. As tempting as it might be, please refrain from bringing to these meetings bugs and troubleshooting questions. Instead use existing channels to get those questions addressed :-)

**Comment by upnix** [comment]
I'm sorry I missed this - I'm looking forward to the recording.

**Reply by mmoatts** [reply]
@upnix , you can find the virtual white board we used [here](https://miro.com/app/board/uXjVKtX5Pu4=/?moveToWidget=3458764596746128023&cot=10) and the recording information in the green box on the right of that board. The next meeting is a week from today. More information on all the upcoming events can be found [here](https://lu.ma/calendar/manage/cal-ZJtdnzAdURyouM7). Looking forward to seeing you at future events!
